src/components/App/filters/filters.js

```diff
@@ -68,3 +76,231 @@ export function formatVersion (string) {
+      translationId = 'filters.asset-policies-verbose-filter.base-atomic-swap'
+      break
+    }
+    // eslint-disable-next-line max-len
+    case ASSET_POLICIES_VERBOSE[ base.xdr.AssetPolicy.canBeQuoteInAtomicSwap().value ]: {
+      translationId = 'filters.asset-policies-verbose-filter.quote-atomic-swap'
+      break
+    }
+    default: {
+      translationId = 'filters.asset-policies-verbose-filter.default'
+      break
+    }
+  }
+  return globalize(translationId)
+}
+export function stellarAssetTypesFilter (value) {
+  let translationId
+
+  switch (value) {
+    case STELLAR_TYPES.credit_alphanum4:
+      translationId = 'filters.stellar-asset-types-filter.alphanumeric-4'
+      break
+
+    case STELLAR_TYPES.credit_alphanum12:
+      translationId = 'filters.stellar-asset-types-filter.alphanumeric-12'
+      break
+
+    case STELLAR_TYPES.native:
+      translationId = 'filters.stellar-asset-types-filter.native'
+      break
+
+    default:
+      translationId = 'filters.stellar-asset-types-filter.default'
+      break
+  }
+
+  return translationId
+}
+
+export function limitsRequestStatesStrFilter (value) {
+  let translationId = ''
+  let key = Object.keys(LIMITS_REQUEST_STATES_STR)
+  switch (value) {
+    case key[0]: {
+      translationId = 'filters.limits-request-states-str-filter.limits-update'
+      break
+    }
+    case key[1]: {
+      translationId = 'filters.limits-request-states-str-filter.initial'
+      break
+    }
+    case key[2]: {
+      translationId = 'filters.limits-request-states-str-filter.docs-uploading'
+      break
+    }
+    default: {
+      translationId = 'filters.limits-request-states-str-filter.default'
+      break
+    }
+  }
+  return globalize(translationId)
+}
+
+export function roleTypeVerboseFilter (value) {
+  let translationId = ''
+  switch (value) {
+    case store.getters.kvAccountRoles.general: {
+      translationId = 'filters.role-type-verbose-filter.general'
+      break
+    }
+    case store.getters.kvAccountRoles.usVerified: {
+      translationId = 'filters.role-type-verbose-filter.us-verified'
+      break
+    }
+    case store.getters.kvAccountRoles.usAccredited: {
+      translationId = 'filters.role-type-verbose-filter.us-accredited'
+      break
+    }
+    case store.getters.kvAccountRoles.corporate: {
+      translationId = 'filters.role-type-verbose-filter.corporate'
+      break
+    }
+    case store.getters.kvAccountRoles.notVerified: {
+      translationId = 'filters.role-type-verbose-filter.not-verified'
+      break
+    }
+    default: {
+      translationId = 'filters.role-type-verbose-filter.default'
+      break
+    }
+  }
+  return globalize(translationId)
+}
+
+export function idDocumentsVerboseFilter (value) {
+  let translationId = ''
+  switch (value) {
+    case ID_DOCUMENT_TYPES.passport: {
+      translationId = 'filters.id-documents-verbose-filter.passport'
+      break
+    }
+    case ID_DOCUMENT_TYPES.identity_card: {
```

Comment:
Always use camelcase for naming keys in the object. To existing some situation when you will be using snake case example: CONST[type]. type - come from bekend.

Reply:
done